docs/cheatsheet/cheatsheet.html

```diff
@@ -0,0 +1,163 @@
+<!--
+Copyright 2016 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<!--
+Only renders in Chrome.
+
+cheatsheet.pdf: Save as PDF from Chrome
+cheatsheet.png: Save as image from Chrome using https://chrome.google.com/webstore/detail/full-page-screen-capture/fdpohaocaechififmbbbbbknoalclacl
+-->
+<html>
+<head>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<link rel="stylesheet" type="text/css" href="cheatsheet.css">
+</head>
+<body>
+
+<div class="header">
+<div class="usage">
+<span class="one">EarlGrey().selectElementWithMatcher</span>(<span class="two">ViewMatcher</span>)
+                      .performAction(<span class="three">ViewAction</span>)
+                      .assertWithMatcher(<span class="two">ViewMatcher</span>)
+</pre>
+</div>
+
+<div class="cheatsheet">
+EarlGrey 1.0.0
+</div>
+</div>
+
+<div class="body">
+<ul>
+  <li>View Matchers</li>
+  <li>User Properties
+    <ul>
+      <li>grey_accessibilityID(...)</li>
+      <li>grey_accessibilityLabel(...)</li>
+      <li>grey_accessibilityValue(...)</li>
+      <li>grey_accessibilityTrait(...)</li>
+      <li>grey_accessibilityHint(...)</li>
+      <li>grey_text(...)</li>
+    </ul>
+  </li>
+  <li>UI Properties
+    <ul>
+      <li>grey_minimumVisiblePercent(...)</li>
+      <li>grey_sufficientlyVisible()</li>
+      <li>grey_interactable()</li>
+      <li>grey_notVisible()</li>
+      <li>grey_systemAlertViewShown()</li>
+      <li>grey_enabled()</li>
+      <li>grey_layout(...)</li>
+      <li>grey_nil()</li>
+      <li>grey_notNil()</li>
+    </ul>
+  </li>
+  <li>Hierarchy
+    <ul>
+      <li>grey_ancestor(...)</li>
+      <li>grey_descendant(...)</li>
+    </ul>
+  </li>
+  <li>Class
+    <ul>
+      <li>grey_kindOfClass(...)</li>
+      <li>grey_respondsToSelector(...)</li>
+      <li>grey_conformsToProtocol(...)</li>
+    </ul>
+  </li>
+  <li>Other
+    <ul>
+      <li>grey_keyWindow()</li>
+      <li>grey_firstResponder()</li>
+      <li>grey_accessibilityElement()</li>
+      <li>grey_progress(...)</li>
+      <li>grey_buttonTitle(...)</li>
+      <li>grey_stepperValue(...)</li>
+      <li>grey_sliderValueMatcher(...)</li>
+      <li>grey_pickerColumnSetToValue(...)</li>
+      <li>grey_datePickerValue(...)</li>
+      <li>grey_switchWithOnState(...)</li>
+      <li>grey_closeTo(...)</li>
+    </ul>
+  </li>
+  <li>Object Matchers
+  <ul>
+    <li>grey_allOfMatchers(...)</li>
+    <li>grey_anyOfMatchers(...)</li>
```

Comment:
Could you put in:

For Swift : 
grey_allOfMatchers(...)
grey_anyOfMatchers(...)

For Objective-c:
grey_allOf(...)
grey_anyOf(...)

We have the difference because we were having difficulties using the variadic c-methods in Swift.